base/Dockerfile.rhel

```
@@ -18,5 +18,9 @@ RUN INSTALL_PKGS=" \
     yum install -y --setopt=tsflags=nodocs --setopt=install_weak_deps=False ${INSTALL_PKGS} && \
     ( test -e /usr/bin/python ||  alternatives --set python /usr/bin/python3 ) && \
     yum clean all && rm -rf /var/cache/*
+
+# Enable x509 common name matching for golang 1.15 and beyond.
+ENV GODEBUG=x509ignoreCN=0
```

```suggestion
ENV GODEBUG=${GODEBUG}:x509ignoreCN=0
```
Probably not necessary since this is a base layer?

---

Nothing at the moment, but a reasonable future proofing. I'll add it.